content/articles/okta-identity-provider.md

```diff
@@ -35,42 +35,29 @@ To proceed with configuring login with SSO through Okta, you must:
   <iframe src="https://www.youtube.com/embed/XNJP2gwIIh4?rel=0&modestbranding=1&cc_load_policy=1&cc_lang_pref=en" class="aspect-ratio--object" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
 </div>
 
-## Logging in via Okta
-
-You'll need to [link an Okta organization to your DNSimple account](#linking-an-okta-organization-to-a-dnsimple-account) before your team members can log in via Okta SSO.
-
-1. To log in to DNSimple using Okta, visit [https://dnsimple.com/login](https://dnsimple.com/login).
-1. Click the <label>Sign in using Okta</label> button.
-1. Enter the company's Okta domain and click "Sign in".
-1. If you are prompted for your Okta username and password, enter them.
-1. If your credentials are valid, you will be redirected back to DNSimple and logged in.
-
-If you are not yet a member of the DNSimple account, the account administrator will receive a notification to grant access. Once they have granted access, you will be able to see the DNSimple account's assets.
-
-If you log out of Okta, you will also be logged out of DNSimple.
-
 ## Linking an Okta organization to a DNSimple account
 
-Follow the instructions below to connect DNSimple to your Okta organization.
+You can also follow the instructions in [Okta’s documentation ](https://help.okta.com/oie/en-us/content/topics/apps/apps-add-applications.htm) to connect DNSimple to your Okta organization. 
 
-### Adding the DNSimple App to Okta {#adding-the-dnsimple-app-to-okta}
+### Adding the DNSimple App to Okta {#adding-the-dnsimple-app-to-okta} 
```

Comment:
My instinct here is to delete these screenshots so we don't have to keep up with vis ID changes on their end. 

We could also add a note to this section that the screenshots are current as of March 2023, so it's clear that they may not be the most up to date. I've also linked the relevant Okta documentation in the steps.